internal/static/static_test.go

```diff
@@ -0,0 +1,72 @@
+package static_test
+
+import (
+	"github.com/dogmatiq/configkit"
+	. "github.com/dogmatiq/configkit/internal/static"
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	"golang.org/x/tools/go/packages"
+)
+
+var _ = Describe("func FromPackages()", func() {
+	When("multiple Dogma applications exist in the packages", func() {
+		It("returns all Dogma application configurations", func() {
+			cfg := packages.Config{
+				Mode: packages.LoadAllSyntax,
+				Dir:  "testdata/multiple-apps",
```

Comment:
I am a little wary of this "multiple-apps" test vector. It definitely covers the case of loading multiple apps, but it looks like it's also being used to cover a few other specific cases such as the difference between when an application key is specified as a constant vs a literal. 

In my experience with these kinds of tests (those where we load large test data and/or compare to golden files) it's been a good idea to keep the tests as focused as possible. Otherwise, when something does fail it can be very difficult to know which aspect is actually causing the problem.

Reply:
[This commit](https://github.com/dogmatiq/configkit/pull/118/commits/79bfd311c5667c114ca84404d70cd7e661e1b7dd) now separates every singly feature tested into a separate test and separate directory within `testdata` directory. Hopefully
this should be a good start for more granular tests.